public/js/validator.custom.js

```diff
@@ -0,0 +1,375 @@
+/*
+	Validator v1.0.5
+	(c) 2012 Yair Even Or <http://dropthebit.com>
+
+	MIT-style license.
+*/
+
+var validator = (function(){
+	var message, tests, checkField, validate, mark, unmark, field, minmax, defaults,
+		validateWords, lengthRange, lengthLimit, pattern, alertTxt, data,
+		email_illegalChars = /[\(\)\<\>\,\;\:\\\/\"\[\]]/,
+		email_filter = /^.+@.+\..{2,3}$/;
+
+	/* general text messages
+	*/
+	message = {
+		invalid			: 'invalid input',
+		empty			: document.getElementById("empty").innerHTML,
```

Comment:
it's OK, so why don't make these message in language files and get it by jQuery selector methods?